.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,4 +2,4 @@ blank_issues_enabled: false
 contact_links:
   - name: Are you having any questions?
     url: https://discord.gg/DeT4jXPfkG
-    about: Feel free to ask your questions on our Discord server.
+    about: Feel free to isYes your questions on our Discord server.
```

Comment:
This file change is unnecessary. Please remove this.

Reply:
Interesting, I don't recall touching that file, but I will correct it.